source/sdk/lang/Exception.ooc

```diff
@@ -34,6 +34,8 @@ exceptionStack := ThreadLocal<Stack<StackFrame>> new()
 _exception := ThreadLocal<Exception> new()
 _EXCEPTION: Int = 1
 
+stackFramesToDelete := Stack<StackFrame> new()
```

Comment:
I know this file was already a mess, but, does this really need to be a global thing? Could it be put inside a class and made static, perhaps?